recipes/plugins_example.ipynb

```diff
@@ -188,7 +180,7 @@
     "from kili.plugins import PluginCore\n",
     "from typing import Dict\n",
```

Comment:
Also should probably add `List, Optional`

Reply:
on it